app/routes/task_routes.py

```diff
@@ -1 +1,144 @@
-from flask import Blueprint
\ No newline at end of file
+from datetime import datetime
+
+from flask import request, Blueprint, make_response, abort
+from sqlalchemy import asc, desc
+
+from app import db
+from app.models.task import Task
+from app.slack.slack_client import SlackClient
+from app.slack.slackmessage import SlackMessage
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+slack_client = SlackClient()
+
+
+@tasks_bp.post("")
+def create_task():
+    request_body = request.get_json()
+    if "title" not in request_body.keys():
+        abort(make_response({"details": "Invalid data"}, 400))
+    title = request_body["title"]
+    if "description" not in request_body.keys():
+        abort(make_response({"details": "Invalid data"}, 400))
+    description = request_body["description"]
+    new_task = Task(title=title, description=description, completed_at=None)
+    db.session.add(new_task)
+    db.session.commit()
```

Comment:
Another great candidate for refactoring to use a `create_model` helper function so that this route is more concise.

When writing route logic, we might ask ourselves questions like, "Does this route need to know how to process a request, handle potential errors in the request body, create an instance of the object, and add/commit it to the DB?" 

We can make our routes concise and readable by using helper methods